index.bs

```diff
@@ -5343,27 +5343,35 @@ to obtain all of the above security benefits. However, one notable use case that
 next section.
 
 
-### Considerations for Self and None Attestation Types and Ignoring Attestation ### {#sctn-no-attestation-security-attestation}
+### Limitations of Guarantees Provided by Attestation ### {#sctn-attestation-guarantee-limitations}
 
-When [[#registering-a-new-credential|registering a new credential]], the [=[WRP]=] MAY choose to accept an [=attestation
-statement=] of [[#sctn-attestation-types|type]] [=self attestation|Self=] or [=None=], or to not verify
-the [=attestation statement=]. In all of these cases the [=[RP]=] loses much of benefit (3) listed above, but retains the other
-benefits.
-
-In these cases it is possible for a [=man-in-the-middle attack|man-in-the-middle attacker=] - for example, a malicious [=client=]
-or script - to replace the [=credential public key=] to be registered, and subsequently tamper with future [=authentication
-assertions=] [=scoped=] for the same [=[RP]=] and passing through the same attacker. Accepting these [[#sctn-attestation-types|types]]
-of [=attestation statements=] therefore constitutes a [=leap of faith=]. In cases where [=registration=] was accomplished
-securely, subsequent [=authentication=] [=ceremonies=] remain resistant to [=man-in-the-middle attacks=], i.e., benefit (3) is
-retained. Note, however, that such an attack would be easy to detect and very difficult to maintain, since any [=authentication=]
-[=ceremony=] that the same attacker does not or cannot tamper with would always fail.
-
-The [=[RP]=] SHOULD consider the above in its threat model when deciding its policy on what [[#sctn-attestation-types|attestation
-types]] to accept or whether to ignore [=attestation=].
+[INFORMATIVE]
 
-Note: The default [[#sctn-attestation-types|attestation type]] is [=None=], since the above issues will likely not be a major
-concern in most [=[RPS]=]' threat models. For example, the [=man-in-the-middle attack=] described above is more difficult than a
-[=man-in-the-middle attack=] against a [=[RP]=] that only uses conventional password authentication.
+When [[#registering-a-new-credential|registering a new credential]], the [=attestation statement=] might allow the [=[WRP]=] to,
+by unspecified procedures, derive guarantees about various qualities of the [=authenticator=], e.g., what kind of
+[=authenticator=] is being used, or how [=credential private keys=] are stored and protected. However, it is important to note
+that the [=attestation statement=] provides no way for the [=[RP]=] to verify that the [=attestation statement=] was generated by
+the [=authenticator=] the user intends, and not by a [=man-in-the-middle attack|man-in-the-middle attacker=], e.g., malicious code
+injected into [=[RP]=] script. The [=[RP]=] must therefore employ other means, e.g., TLS and related technologies, to protect the
+[=attestation statement=] from [=man-in-the-middle attacks=].
+
```

Comment:
suggestion for above parag:

When [[#registering-a-new-credential|registering a new credential]], the [=attestation statement=], if present, may allow the [=[WRP]=] to derive assurances about various [=authenticator=] qualities. For example, the
[=authenticator=] model, or how it stores and protects [=credential private keys=]. However, it is important to note
that an [=attestation statement=], on its own, provides no means for a [=[RP]=] to verify that an [=attestation object=] was generated by
the [=authenticator=] the user intended, and not by a [=man-in-the-middle attack|man-in-the-middle attacker=]. For example, malicious code
injected into [=[RP]=] script. The [=[RP]=] must therefore rely on other means, e.g., TLS and related technologies, to protect the
[=attestation object=] from [=man-in-the-middle attacks=].

Reply:
Thanks! I also added:
s/For example, malicious code/For example, such an attacker could use malicious code/

Reply:
nice :)